test-server/src/Theme.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class Theme extends Component {
+  componentDidMount() {
+    const presentTheme = localStorage.getItem("theme");
+    const presentFont = localStorage.getItem('fonts')
+    if (presentTheme) {
+      document.querySelector('html').style.setProperty('--bg-color',`${presentTheme}`)
+    }
+    if (presentFont)
+      {document.querySelector('html').style.setProperty('--heading-color',`${presentFont}`)}
+  }
+  submitbackground(e)
+  {
+    console.log(e.target.value)
+    this.onChange(e.target.value)
+  }
+  submitfont(e){
+    localStorage.setItem('fonts',e.target.value)
+    document.querySelector('html').style.setProperty('--heading-color',`${e.target.value}`)
+  }
+
+  onChange(e) {
+    localStorage.setItem('theme',e)
+    const val = document.querySelector('html')
+    val.style.setProperty('--bg-color',`${e}`)
+    console.log(val)
+  }
+
+  render() {
+    return (
+      <div className="App"  >
+      <form action="/action_page.php" onChange = {(e) => this.submitbackground(e)} >
```

Comment:
@nikhil7sunny  Why is there a php script called here?

Reply:
I will remove  the php script ,i forgot to remove it earlier. I am adding my codepen link
here : https://codepen.io/__nikhil7sunny__/pen/yLYmzzJ

Reply:
@nikhil7sunny Codepen link isn't working. Please check.